controllers/research/upcomingEventController.js

```diff
@@ -109,11 +109,27 @@ const deleteUpcomingEventByID = (req, res) => {
         .catch((err) => sendError(res, err));
 };
 
+// new function which will trigger the organized property
+
+const isEventOrganized = (req,res) => {
+    const id = req.query.id;
```

Comment:
There is no such object as `req.query.id` unless this function is called through an endpoint